.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,18 +19,20 @@ from the implementation proposal in the referenced issues.
 
 **Notes for Reviewers:**
 <!--
-How should your reviewers approach this pull request?
-Any special requests or questions for specific reviewers?
+- Motivate briefly why it is implemented this way, if that deviates
+  from the implementation proposal in the referenced issues.
+- How should your reviewers approach this pull request?
+- @mention reviewers with special requests or questions for them
 -->
 
 ...
 
 **Release Notes: _(optional)_**
 <!--
 Any notes that we need to include in the Release Notes for the next release.
-What are the functional or behavioral changes? API Changes? New features?
-Any changes in configuration? Added/changed/removed flags?
-Are there any database migrations required?
+- Always mention changes in API, database models, configuration options or defaults.
+- Are there any database migrations required?
+- What are the functional or behavioral changes?
```

Comment:
Let's enforce a bullet-point format here so that we can copy that to release notes.

Also let's add guidelines for the sentence, like we do with commit messages. I.e. I think this should be past tense, like "Added support for Basic Station"

Reply:
Handing this part over to @rvolosatovs, then he can make the changes required for #178.

Reply:
I'm not sure I will have time to thoroughly look into this tomorrow, since I have higher priority things to work on.
For now a bullet-point format, as @johanstokking mentioned in https://github.com/TheThingsNetwork/lorawan-stack/pull/322/files#r267226449 should be good enough
I will update the guidelines if necessary as part of #178

Reply:
Resolved as requested